### ISSUE TITLE
[ENHANCEMENT] 

### ISSUE THREAD

**Issue by LittleVidoskim** [Elytrium/LimboAuth#111]
When player non license use /premium confirm, he cant join, Please add license check or force disable premium

**Comment by Kamillaova** [comment]
> Please add license check
How? This is impossible.
> force disable premium
Use /forcechangepassword

**Comment by Kamillaova** [comment]
@LittleVidoskim ??

**Comment by LittleVidoskim** [comment]
> > Please add license check
> 
> How? This is impossible.
> 
> > force disable premium
> 
> Use /forcechangepassword
> 
> @LittleVidoskim

Why? Simple search player nick on minecraft auth

**Comment by Kamillaova** [comment]
> Why? Simple search player nick on minecraft auth
 
https://github.com/Elytrium/LimboAuth/blob/master/src/main/java/net/elytrium/limboauth/Settings.java#L76

**Comment by Kamillaova** [comment]
tldr - set online mode need auth to false
